releasenotes/notes/fix-circuitsampler-bindings-for-listop-5f98df655b3151e4.yaml

```diff
@@ -0,0 +1,10 @@
+---
+fixes:
```

Comment:
The text makes its sound more like a feature enhancement than a bug fix. Should this be a feature rather than a bug fix?

Reply:
Yes you could interpret it as both. I originally had "bugfix" in mind because this missing feature broke the gradients in some setting, but I'm fine labeling this as "feature", too 🙂